diff --git a/src/PIL/Image.py b/src/PIL/Image.py
index c2216e27a..0992fb528 100644
--- a/src/PIL/Image.py
+++ b/src/PIL/Image.py
@@ -106,7 +106,7 @@ try:
     # and should be considered private and subject to change.
     from . import _imaging as core
 
-    if __version__ != getattr(core, "PILLOW_VERSION", None):
+    if __version__ != getattr(core, "PILLOW_VERSION", None).strip('"'):
         raise ImportError(
             "The _imaging extension was built for another version of Pillow or PIL:\n"
             f"Core version: {getattr(core, 'PILLOW_VERSION', None)}\n"
